packages/mobile/src/screens/change-email-screen/SubScreens.tsx

```diff
@@ -72,7 +75,14 @@ export const NewEmailSubScreen = () => {
         title={messages.changeYourEmail}
         description={messages.newEmailHelp}
       />
-      <HarmonyTextField name='email' label={messages.newEmail} />
+      <HarmonyTextField
```

Comment:
curious if we could use any existing email logic, but really not a big deal if so

Reply:
Responded in the other pr lmk what you think